installer/pkg/components/registry-facade/certificate.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package registryfacade
+
+import (
+	"fmt"
+	"time"
+
+	certmanagerv1 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
+	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
+
+	"github.com/gitpod-io/gitpod/installer/pkg/common"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func certificate(ctx *common.RenderContext) ([]runtime.Object, error) {
+	oneYear := &metav1.Duration{Duration: time.Hour * 24 * 365}
+
+	return []runtime.Object{&certmanagerv1.Certificate{
+		TypeMeta: common.TypeMetaCertificate,
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      common.RegistryFacadeTLSCertSecret,
+			Namespace: ctx.Namespace,
+			Labels:    common.DefaultLabels(Component),
+		},
+		Spec: certmanagerv1.CertificateSpec{
+			Duration:   oneYear,
```

Comment:
Possibly not relevant for this, but as we're using Cert Manager for our internal certs, what `Duration` should we use for them? I think we're using 6 months for some and 1 year for others (including this one) - and does the `Duration` even matter?

Reply:
I think that more important than the duration value itself, we need to make sure the renewal works and the components pick the new cert.
I would even set this for an hour until we release an official version.

Reply:
Good point. I'll raise a ticket separate to this PR with that in